### ISSUE TITLE
English - Create <p> paragraph element

### ISSUE THREAD

**Issue by kathyoleary** [kevinbrianfahy/LMHT#144]
In order to learn more elements,
As an LMHT user,
I want to be able to look up and learn about the paragraph element.

Acceptance Criteria:
The english text sections of the standard element page will be completed for `<p>`.

Given any LMHT user,
I want to select  and find the `<p>` element page
Where I can learn about the element in english.

**Comment by kathyoleary** [comment]
```
<p>
```

English base: Paragraph

Definition:   The paragraph is a block level element (as opposed to an inline element) meaning it occupies the entire space of its container or parent element.  Therefore, it can contain inline elements within it.  Paragraphs are blocks of related text separated from other paragraphs by blank space and usually each paragraph is started with an indentation to mark the start of a new paragraph.  Paragraphs are the building blocks of reports, articles, stories, books or any other type of written documentation. 

How to Remember:  Paragraphs are the building blocks of written documentation just as bricks are the building blocks of buildings.  Put enough connected blocks together and you build a document - put enough bricks together and you have a building.

Code example:
    `<p>Four score and seven years ago our fathers brought forth on this continent, a new nation, conceived in Liberty, and dedicated to the proposition that all men are created equal. </p>`

**Comment by JenEEsquivel** [comment]
:+1:  Possible edit:  "Put enough connected paragraph blocks together and you build a document; put enough bricks together and you have a building.

**Comment by kathyoleary** [comment]
Ok, 
Ran into a couple of interesting things.  We currently do not have the code example box set to word wrap so anything that goes past the end of the box just keeps running off the end of the web page in a lighter color of text.  

![screen shot 2016-03-16 at 1 53 54 pm](https://cloud.githubusercontent.com/assets/15365377/13825009/9a6b8474-eb7e-11e5-8ecb-0f550086203d.png)

So I applied word wrap, but then it doesn't indent properly.  Not only that but do we want it to show this way?  How much will this mess up more complex coding by just wrapping wherever?

![screen shot 2016-03-16 at 1 37 16 pm](https://cloud.githubusercontent.com/assets/15365377/13825024/af72ea4c-eb7e-11e5-84c2-86e7bc6033b2.png)

So, for now, using a shorter code example, and then opening this up as a new issue to address on what we want to do with this.
